### PR TITLE
Pricing page enhancements

### PR DESCRIPTION
**Pull request by shawyu-okta**
## Description:
- **What's changed?**
  * Flows the features list to a single column at mobile viewport widths
  * Updates font on FAQ question to be Proxima Nova
  * Ensures check icons in editions matrix do not shrink at smaller viewport widths

### Resolves:

* [OKTA-355569](https://oktainc.atlassian.net/browse/OKTA-355569)

### CODE REVIEW

File: packages/@okta/vuepress-theme-prose/index.js [hunk 1]
```diff
@@ -14,9 +14,9 @@ module.exports = (options, ctx) => {
       $page.fullHeaders = resolveHeaders($page);
 
       // Set to true to enable the redesign in development
-      const redesign = process.env.REDESIGN || false;
+      //const redesign = process.env.REDESIGN || false;
       // Respect the redesign flag unless this is a production build
-      $page.redesign = !ctx.isProd && redesign;
+      $page.redesign = true;
```
Comment (shawyu-okta) [inline packages/@okta/vuepress-theme-prose/index.js:19]: Note to self to revert before merge.

**Comment by shawyu-okta** [comment]
Netlify deploy: https://5ff34aee5185de5468bcec37--reverent-murdock-829d24.netlify.app/pricing/

**Review by TiffanyNg-okta — APPROVED** [review]

**Comment by haishengwu-okta** [comment]
Two questions

Screen Shot 2021-01-05 at 8 03 06 AM
Screen Shot 2021-01-05 at 8 02 59 AM

**Review by haishengwu-okta — APPROVED** [review]
👍 with additional questions.

**Comment by shawyu-okta** [comment]
@haishengwu-okta good eye -- I looked at the existing pricing page and your points are good ones. I'll make some touch ups to the spacing and alignment before merging

**Pull request merged**